### PR TITLE
operator t8c-certified (42.54.0-beta.1)

### PR DESCRIPTION
**Pull request by turbodeploy**
operator t8c-certified (42.54.0-beta.1)

### CODE REVIEW

File: operators/t8c-certified/42.54.0-beta.1/manifests/t8c-certified.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    description: Turbonomic Workload Automation for Multicloud simultaneously optimizes performance, compliance, and cost in real-time. Workloads are precisely resourced, automatically, to perform while satisfying business constraints.
+    repository: https://github.com/turbonomic/t8c-install/tree/master/operator
+    support: Turbonomic, Inc.
+    olm.skipRange: '>=42.53.0-beta.1 <42.54.0-beta.1'
+  name: t8c-operator.v42.54.0-beta.1
```
Comment (acornett21) [inline operators/t8c-certified/42.54.0-beta.1/manifests/t8c-certified.clusterserviceversion.yaml:26]: The name here seems incorrect, I believe it should be `t8c-certified`.

Comment (turbodeploy) [reply operators/t8c-certified/42.54.0-beta.1/manifests/t8c-certified.clusterserviceversion.yaml:26]: But it has never been named that way.   From Day 1 this is the naming convention.

Comment (acornett21) [reply operators/t8c-certified/42.54.0-beta.1/manifests/t8c-certified.clusterserviceversion.yaml:26]: I have added the appropriate label to the pr.

**Pull request merged**